internal/pkg/cli/env_init.go

```diff
@@ -156,6 +156,7 @@ func (o *initEnvOpts) Execute() error {
 	if err != nil {
 		return fmt.Errorf("get environment struct for %s: %w", o.EnvName, err)
 	}
+	env.Prod = o.IsProduction
```

Comment:
can we add a unit test for this? where the GetEnvironment returns an env.Prod = false and we set it to true